Index.html

```diff
@@ -30,109 +36,156 @@ <h2 class="connect">LET'S CONNECT</h2>
-      <h2 class="connect">LET'S CONNECT</h2>
-      <ul class="icons subIcons">
-        <li class="gitHub"><img src="Assests/Vectorgithub.svg"></li>
-        <li><img src="Assests/Linkedin iconlinked.svg"></li>
-        <li><img src="Assests/Vectorangerllist.svg"></li>
-        <li class="twitter bird"><img src="Assests/Vectortwitter.svg"></li> 
-        <li class="medium"><img src="Assests/Vectormedium.svg"></li> 
-      </ul>
-      <button class="secondButton">Get my resume</button>
-      <ul class="abilities">
-        <li class="languages"><aside class="textOne">Languages</aside><img class="droplet down" src="Assests/preview.png">
-          <ul class="subLanguages">
-            <li class="iconw"><img class="image" src="Assests/Ellipse 1jsLogo.svg">JavaScript</li>
-            <li class="iconw"><img class="image" src="Assests/Ellipse 1htmlLogo.svg">HTML</li>
-            <li class="iconw"><img class="image" src="Assests/Ellipse 1css.svg">CSS</li>
-          </ul>
-        </li>
+      <div class="abouts">
+        <div class="about1">
+          <h1 class="introTitle introTwo">About Myself</h1>
+          <p class="textIntro">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          <h2 class="connect">LET'S CONNECT</h2>
+          <ul class="icons subIcons">
+            <li class="gitHub"><img src="Assests/Vectorgithub.svg"></li>
+            <li><img src="Assests/Linkedin iconlinked.svg"></li>
+            <li><img src="Assests/Vectorangerllist.svg"></li>
+            <li class="twitter bird"><img src="Assests/Vectortwitter.svg"></li> 
+            <li class="medium"><img src="Assests/Vectormedium.svg"></li> 
+          </ul>  
+          <button class="secondButton">Get my resume</button>      
+        </div>
+        <div class="about2">
+        <ul class="abilities">
+          <li class="languages"><aside class="textOne">Languages</aside><img class="droplet down" src="Assests/preview.png">
+            <ul class="subLanguages">
+              <li class="iconw"><img class="image" src="Assests/Ellipse 1jsLogo.svg">JavaScript</li>
+              <li class="iconw"><img class="image" src="Assests/Ellipse 1htmlLogo.svg">HTML</li>
+              <li class="iconw"><img class="image" src="Assests/Ellipse 1css.svg">CSS</li>
+            </ul>
+          </li>
+        </ul>
         <li class="frameworks languages">Frameworks<img class="droplet side" src="Assests/droplet.png"></li>
         <li class="skills languages">Skills<img class="droplet sideTwo" src="Assests/droplet.png"></li>
-      </ul>
+      </div>
+    </div>

```

Comment:
[OPTIONAL] This section is overflowing with the contact one 🤔 now this issue is only present when I test directly on a phone size of `375` px but it's not there when I resize responsively, take a look 👇🏽 

![Screenshot 2023-05-07 at 01 36 21](https://user-images.githubusercontent.com/45739963/236659767-28e0c84b-20bf-475d-9ed1-2bd8b159acbb.png)